sql/total-active-cases-table.sql

```diff
@@ -0,0 +1,28 @@
+with t as (
+    select
+        classification,
+        case when
+            propertytype is null then 'All' else propertytype end as propertytype,
+        case when 
+            (court = any('{
+            Bronx County Civil Court,
+            Kings County Civil Court,
+            New York County Civil Court,
+            Queens County Civil Court,
+            Richmond County Civil Court,
+            Redhook Community Justice Center,
+            Harlem Community Justice Center
+        }')) then 'NYC' 
+        else 'Outside NYC' end as region,
+        case when
+            fileddate < '2021-03-21' then 'Issued Prepandemic'
```

Comment:
OMG @samaratrilling! This should be `2020-03-21` not 2021, right? Wow I've literally been staring at this for 40 minutes trying to find out why your table shows different pre-pandemic case counts from the cumulative chart...